### PR TITLE
CGSS1 harvester update

### PR DESCRIPTION
**Pull request by davidcordeiro-deimos**
### Features:
Regarding issue #157 
* Remove unwarranted metadata fields

Regarding issue #168 
* Restructure of the resource parsing function

- [X] includes new  features
- [ ] includes tests covering changes
- [ ] includes updated documentation
- [X] includes user-visible changes
- [ ] includes bugfix

Please [X] all the boxes above that apply

### CODE REVIEW

File: ckanext/nextgeossharvest/harvesters/cgss1.py [hunk 1]
```diff
@@ -295,26 +295,27 @@ def _parse_content(self, content_str):
...
-        parsed_content['Collection'] = str(collection)
-        parsed_content['notes'] = parsed_content['description']
         parsed_content['identifier'] = self._parse_identifier(identifier)
         parsed_content['name'] = self._parse_name(identifier)
-        parsed_content['filename'] = self._parse_filename(identifier)
         parsed_content['spatial'] = json.dumps(
             self._bbox_to_geojson(self._parse_bbox(content)))
         if 'groups' in self.source_config:
             parsed_content['groups'] = self.source_config['groups']
-        #parsed_content['is_output'] = True
```
Comment (georgiana-b) [inline ckanext/nextgeossharvest/harvesters/cgss1.py]: If these really are output datasets it would be important to mark them as such.

Comment (georgiana-b) [reply ckanext/nextgeossharvest/harvesters/cgss1.py]: @davidcordeiro-deimos Are these output datasets?

Comment (davidcordeiro-deimos) [reply ckanext/nextgeossharvest/harvesters/cgss1.py]: @georgiana-b According to @joaandrade and Koushik, the products from this data provided are not to be considered an output.

Comment (georgiana-b) [reply ckanext/nextgeossharvest/harvesters/cgss1.py]: Ok, in that case we can remove that line to avoid confusion and then we can merge this.

**Comment by georgiana-b** [comment]
Looks like the merge removed that confusing commented line too so this is ready to go now.

**Pull request merged**